src/EnergyPlus/WindowComplexManager.cc

```diff
@@ -2426,9 +2446,11 @@ namespace WindowComplexManager {
                 } else {
                     Phi = Psi;
                 }
-            } else {
-                assert(false);
+                break;
             }
+            default:
+                assert(false);
```

Comment:
Should this `assert(false)` be replaced by an error message?

Reply:
No. Assert(false) is for impossible situations/developer errors, which this is.

Reply:
Got it. Thank you.